### ISSUE TITLE
NASCAR the game: Inside Line

### ISSUE THREAD

**Issue by coconuty4765** [RPCS3/rpcs3#16334]
### Quick summary

when i try to do a green flag pit stop at Kansas (might be everywhere) the game freezes when the gas can is pulled away
![image](https://github.com/user-attachments/assets/704ff401-7b01-4dcb-a444-754dcba90a42)

### Attach a log file

[f2l-1,f2l-2,f2l-3.zip](https://github.com/user-attachments/files/17839294/f2l-1.f2l-2.f2l-3.zip)

### Attach capture files for visual issues

![image](https://github.com/user-attachments/assets/d51b7ebe-011b-4875-be2a-d1db050ccb8f)

### System configuration

Windows 11 x64
11th Gen Intel Core i7-11800H @ 2.30 GHz
Nvidia GeForce RTX 3050Ti Laptop GPU
GeForce Game Ready Drivers Version: 566.03

**Comment by coconuty4765** [comment]
wrong zip, i for some reason cant upload the log its saying im not the owner

**Comment by coconuty4765** [comment]
and also the car is black for some reason

**Comment by coconuty4765** [comment]
how?

**Comment by coconuty4765** [comment]
but now its freezing here
![image](https://github.com/user-attachments/assets/d0dfcae1-46a4-4815-9729-269e5aef52f1)

**Comment by SanokMaynik** [comment]
Can confirm, it crashes at every track when making a pit stop, game runs fine otherwise

[RPCS3.log](https://github.com/user-attachments/files/18286521/RPCS3.log)

**Comment by SanokMaynik** [comment]
Update: game seems to not crash if you select "Disable Asynchronous Memory Manager"

**Comment by LG771** [comment]
> Update: game seems to not crash if you select "Disable Asynchronous Memory Manager"

Unfortunately still crashes for me even with the option selected.
![Screenshot 2025-01-05 095014](https://github.com/user-attachments/assets/f8fa58c8-c8ee-4dff-869c-49c5aea1ee5a)
![Screenshot 2025-01-05 094851](https://github.com/user-attachments/assets/52643a90-a47c-45dd-b10b-c5620e4706f2)
![Screenshot 2025-01-05 094941](https://github.com/user-attachments/assets/807b4a1f-50e8-4029-9d00-eaeceb76a9dc)
[RPCS3.log](https://github.com/user-attachments/files/18327588/RPCS3.log)

**Comment by coconuty4765** [comment]
yeah, its still crashing for me. 
[RPCS3.log](https://github.com/user-attachments/files/18340894/RPCS3.log)
![image](https://github.com/user-attachments/assets/cec0b51c-11b4-4e5c-9654-09258a14ee7d)

**Comment by SanokMaynik** [comment]
Yeah ok i just went back to run a full race and i got 2 pit stops without crashing but it crashed on the 3rd one, so it's definitely not an actual fix

**Comment by LG771** [comment]
Has anyone tried it yet?

**Comment by digant73** [comment]
@coconuty4765, @SanokMaynik is that still an issue?

**Comment by Livi-The-Queen** [comment]
No changes on 0.0.38-18263-3c401e7b Alpha | master

[RPCS3.log](https://github.com/user-attachments/files/23176033/RPCS3.log)

**Comment by digant73** [comment]
please an update on this issue

**Comment by digant73** [comment]
please an update on this issue

**Comment by kanesean19-art** [comment]
Still an issue in the most recent build sadly, tried various changes but doesn't get through the animation without crashing

**Comment by kanesean19-art** [comment]
First race I've done today I was able to make a green flag stop without a crash, backed out to another race and pit on Lap 1 and it crashed at the same point as everyone else and every pit stop after re-launching
